# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/visibility.rs
# Commit: 827ff7c5f5f4
# Full Hash: 827ff7c5f5f4f74aad42f25ccae7b314369c0b11
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-05-25 03:42:54
# Regressor Bug: 1769963
# File Overlap Count: 1
# Description:
#   Bug 1769963 - Fix backdrop-filter inside complex transform with CSS filter chain r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D147124
# ==============================================================================

diff -r aac2eb307b0b -r 827ff7c5f5f4 gfx/wr/webrender/src/visibility.rs
--- a/gfx/wr/webrender/src/visibility.rs	Tue May 24 19:29:01 2022 +0000
+++ b/gfx/wr/webrender/src/visibility.rs	Tue May 24 19:30:19 2022 +0000
@@ -360,6 +360,7 @@
                 &frame_state.surface_stack,
                 &mut frame_state.composite_state,
                 &mut frame_state.gpu_cache,
+                &mut frame_state.scratch.primitive,
                 is_root_tile_cache,
                 surfaces,
             );